app/uk/gov/hmrc/gform/sharedmodel/form/UserData.scala

```diff
@@ -17,15 +17,17 @@
 package uk.gov.hmrc.gform.sharedmodel.form
 
 import play.api.libs.json.{ Json, OFormat, OWrites, Reads }
-import uk.gov.hmrc.gform.sharedmodel.{ Obligations }
+import uk.gov.hmrc.gform.sharedmodel.Obligations
 import play.api.libs.functional.syntax._
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.EmailParameters
 
 case class UserData(
   formData: FormData,
   formStatus: FormStatus,
   visitsIndex: VisitIndex,
   thirdPartyData: ThirdPartyData,
-  obligations: Obligations
+  obligations: Obligations,
+  emailParameters: EmailParameters
```

Comment:
`emailParameters ` shouldn't be part of `UserData` nor `Form`. Use `SubmissionData` for passing data from frontend to backend. See `submissionWithPdf` in SubmissionService.